PROTO_tests/tests/security_test.py

```diff
@@ -0,0 +1,22 @@
+import platform
+from os.path import expanduser
+
+from arkouda import security
+
+
+class TestSecurity:
+    def test_generate_token(self):
+        assert 32 == len(security.generate_token(32))
+        assert 16 == len(security.generate_token(16))
+
+    def test_get_home(self):
+        assert expanduser("~") == security.get_home_directory()
+
+    def test_get_username(self):
+        assert security.get_username() in security.username_tokenizer[platform.system()](
+            security.get_home_directory()
+        )
+
+    def testGetArkoudaDirectory(self):
```

Comment:
I would just match the test name to the style of the others here, otherwise looks good

Reply:
Updated